docs/src/content/components/drawer.mdx

```diff
@@ -7,13 +7,17 @@ import Drawer from '@pluralsight/ps-design-system-drawer'
 
 # Drawer
 
-<TableOfContents {...props} />
+<Intro>
+  The Drawer can be used as an uncontrolled or controlled component. It consist
+  of two subcomponents Summary and Details. The top-level component Drawer is headless.
+  It's puporse is to supply context and cordinate the subcomponents.
+</Intro>
 
-## Examples
+<TableOfContents {...props} />
 
-### Adaptive size
+## Examples: uncontrolled
```

Comment:
Instead of broad, page-level categories, I think I could understand these better if they were individually marked as un/controlled, if that's helpful information, pertinent to the specific example.